src/getAverage.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * Implement a custom array method getAverage:
```

Comment:
suggestion: function names should start with the verb. Maybe calculateAverage?

Reply:
agree

Reply:
`get` is a verb
But I agree that here we have calculation rather than simple getting